workspaces/adventure-pack/goodies/java/src/integer_digits/goody.json

```diff
@@ -0,0 +1,3 @@
+{
+  "name": "integerDigits()"
```

Comment:
One more thing, I think it would probably be ok to give this a short name like `digits()`. Alternatively, it might be beneficial to call it `digits(int)`, highlighting that the argument is an `int`, since we might want to have a `long` version as well.

Reply:
Comment addressed.